CIScripts/Test/TestConfigurationUtils.ps1

```diff
@@ -322,6 +322,21 @@ function Remove-AllUnusedDockerNetworks {
     }
 }
 
+function Wait-RemoteInterfaceIP {
+    Param ([Parameter(Mandatory = $true)] [PSSessionT] $Session,
+           [Parameter(Mandatory = $true)] [Int] $ifIndex)
+
+    Invoke-Command -Session $Session -ScriptBlock {
+        $Address = Get-NetIPAddress -InterfaceIndex $Using:ifIndex -ErrorAction SilentlyContinue `
+                    | ? AddressFamily -eq IPv4 `
+                    | ? SuffixOrigin -eq "Dhcp"
+        if (!$Address) {
+            $AdapterConfig = Get-WmiObject -Class Win32_NetworkAdapterConfiguration | ? InterfaceIndex -eq $Using:ifIndex
+            $AdapterConfig.RenewDHCPLease() | Out-Null
```

Comment:
We think we should check if it succeeded. Please look here (https://msdn.microsoft.com/en-us/library/aa393064(v=vs.85).aspx). Maybe throw an exception when return value is other than 0.